Troubleshooting.md

```diff
@@ -58,9 +58,17 @@ Be sure to tag Peter and Brian (of the lab) on the new card.
 ```sh
 No sample with identifier «aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa» found
 ```
-This means that specimen information sent from the LIMS to ID3C is out of date. The lab may just be behind in syncing it. We can prompt them to do so if needed.
+or
+```sh
+No sample found with identifier «aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa» or collection identifier «None»
+```
+This means that specimen information sent from the LIMS (Lab Inventory Management System) to ID3C is out of date. This is a time-sensitive problem that will impact the return of testing results to participants.
```

Comment:
Should we add that the lab is also notified that the pipeline is erroring in the slack pipeline alerts channel? We have asked them to attempt resolutoin of these issues (especially when it can be resolved by resyncing LIMS) when they see errors in that channel.